tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/containers/PulsarContainer.java

```diff
@@ -97,9 +98,13 @@ public void start() {
         this.withCreateContainerCmdModifier(createContainerCmd -> {
             createContainerCmd.withHostName(hostname);
             createContainerCmd.withName(getContainerName());
-            createContainerCmd.withEntrypoint(serviceEntrypoint);
+            createContainerCmd.withEntrypoint(serviceEntryPoint);
         });
 
+        if(Boolean.getBoolean("containerLogs")) {
```

Comment:
`containerLogs` is too generic. who knows what does it really mean here?

I would prefer a more specific name `enableSlf4jLogConsumer` and prefixing `pulsar.integration.tests.testcontainers` is even better (to avoid naming conflicts).

`pulsar.integration.tests.testcontainers.enableSlf4jLogConsumer`.

Reply:
will remove it for now, will reconsider it in the future.